demo/src/plugins/dashboard/components/active-edits/index.jsx

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  Table,
+} from '@triniti/admin-ui-plugin/components';
+import delegate from './delegate';
+import selector from './selector';
+import TableRow from './TableRow';
+
+class ActiveEdits extends React.Component {
+    static propTypes = {
+      accessToken: PropTypes.string.isRequired, // eslint-disable-line react/forbid-prop-types
+      collaborationNodes: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
+      handleRequestCollaborationNodes: PropTypes.func.isRequired,
+      location: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+      title: PropTypes.oneOf(['Articles', 'Non-Article Nodes']).isRequired,
+    };
+
+    constructor(props) {
+      super(props);
+      this.handleRequestCollabNodes = this.handleRequestCollabNodes.bind(this);
+    }
+
+    componentDidMount() {
+      this.handleRequestCollabNodes();
+    }
+
+    componentDidUpdate(prevProps) {
+      const { location } = this.props;
+      const { location: prevLocation } = prevProps;
+      /**
+         * If user clicks same nav item they are already on (same pathname but different key)
```

Comment:
@nanetteCodes i think this whole lifecycle method does not apply here, because if they click the dashboard nav item its going to take them to the regular one anyways

Reply:
removed